pkg/reconciler/v1alpha1/clusterbuilder/clusterbuilder_test.go

```diff
@@ -66,7 +66,7 @@ func testClusterBuilderReconciler(t *testing.T, when spec.G, it spec.S) {
 			Generation: initialGeneration,
 		},
 		Spec: v1alpha1.BuilderSpec{
-			Image: clusterImageName,
+			Image:        clusterImageName,
```

Comment:
I wonder if we should start more agressively `go fmt ./..`

Reply:
I agree 100%